core/src/main/resources/db/sql_migration_scripts/1.0.0.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE sample_list LIKE patient_list;
+INSERT sample_list SELECT * FROM patient_list;
+CREATE TABLE sample_list_list LIKE patient_list_list;
+INSERT sample_list_list SELECT * FROM patient_list_list;
+DROP TABLE micro_rna;
+DROP TABLE micro_rna_alteration;
+DROP TABLE mutation_frequency;
+ALTER TABLE sample_list_list CHANGE PATIENT_ID SAMPLE_ID INT(11);
+CREATE TABLE version (VERSION_NUMBER VARCHAR(8));
```

Comment:
Maybe rename table to info with a column DB_SCHEMA_VERSION. We can incorporate other information in the table.